superset/viz.py

```diff
@@ -1186,6 +1186,8 @@ def process_data(self, df, aggregate=False):
                 df = df.rolling(**kwargs).sum()
         elif rolling_type == "cumsum":
             df = df.cumsum()
+        elif rolling_type == "cumperf":
```

Comment:
I was wondering how this is relevant to the `Year to Date` time range?

Reply:
Apologies, 
I included in the pull request 2 changes, instead of only the one related to "Year to Date".
The "cumperf" calculation is a very common in the asset management industry, especially for to the "Year to Date" time range.

Shall I re-submit properly ?
More in general, is there appetite for these kind of enhancements ?

Many Thanks